src/routes/(admin)/tests/test-[type]/[id]/QuestionList.svelte

```
@@ -1,15 +1,23 @@
+	import SelectedQuestionsList from './question-selection/SelectedQuestionsList.svelte';
+
+	let { formData, questions, questionParams, selectedQuestions = [] } = $props();
 	let dialogOpen = $state(false);
+
+	const handleRemoveQuestion = (questionId: number) => {
+		// remove from both IDs and question data
+		$formData.question_revision_ids = $formData.question_revision_ids.filter(
+			(id: number) => id !== questionId
+		);
+		$formData.question_revisions = $formData.question_revisions.filter(
+			(question: any) => question.id !== questionId
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)